packages/flutter/lib/src/material/switch.dart

```diff
@@ -1294,6 +1304,17 @@ class _SwitchPainter extends ToggleablePainter {
     notifyListeners();
   }
 
+  bool get isCupertino => _isCupertino!;
```

Comment:
I think this is OK although the implementation might eventually need an enum so that other platforms could be specified.